armi/reactor/converters/tests/test_uniformMesh.py

```diff
@@ -79,6 +79,26 @@ def test_blueprintCopy(self):
         # ensure that the assemblies were copied over
         self.assertTrue(converted.assemblies, msg="Assembly objects not copied!")
 
+    def test_makeAssemWithUniformMesh(self):
+
+        sourceAssem = self.r.core.getFirstAssembly(Flags.FUEL)
+        sourceAssem[2].p["xsType"] = "B"
+        sourceAssem[4].p["ztop"] = 176.0
+        self.converter._computeAverageAxialMesh()
+        newAssem = self.converter.makeAssemWithUniformMesh(
+            sourceAssem, self.converter._uniformMesh
+        )
+
+        for newB, sourceB in zip(newAssem.getBlocks(), sourceAssem.getBlocks()):
+            self.assertEqual(
+                newB.p["xsType"],
+                sourceB.p["xsType"],
+            )
+            self.assertEqual(
+                newB.p["xsType"],
+                sourceB.p["xsType"],
+            )
```

Comment:
These assertions are the same?

Reply:
Also, since the test reactor in `armi/tests/refSmallReactorBase.yaml` is already axially uniform, I'd be curious to know how this test works for the test reactor in `armi/tests/detailedAxialExpansion`.

Reply:
@albeanth the `setUpClass` method for `TestUniformMeshComponents` perturbs the mesh of one of the assemblies.